README.md

```diff
@@ -1,59 +1,69 @@
+## Usage
+A repository has been created on the AWS account *'cloud-platform-aws'* called *'754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/cluster-backup-checker'*. 
 
-A repository has been created on the AWS account *'cloud-platform-aws'* called *'arn:aws:ecr:eu-west-2:754256621582:repository/ƒ'*.
+You can build, tag and push to your own ECR repository and use it to run as a cronjob or as a pod in your own namespace. 
 
-### Building, tagging and pushing to ECR
-1) Retrieve the `docker login` command that you can use to authenticate your Docker client to your registry:
+1) Login to Amazon ECR. 
 
-```aws ecr get-login --no-include-email --region eu-west-2```
+```
+$(aws ecr get-login --no-include-email --region eu-west-2)
+```
+
+You should have AWS credentials(access_key_id and secret_access_key) to login to ECR.
 
-2) Run the `docker login` command that was returned in the previous step.
+2) Build your Docker image using the following command.
 
-3) Build your Docker image using the following command.
+```
+docker build -t cloud-platform/cluster-backup-checker .
+```
 
-```docker build -t cloud-platform/cluster-backup-checker .```
+3) After the build completes, tag your image so you can push the image to the AWS ECR repository:
 
-4) After the build completes, tag your image so you can push the image to this repository:
+```
+docker tag cloud-platform/cluster-backup-checker:latest 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/cluster-backup-checker:<VERSION>
+```
 
-```docker tag cloud-platform/cluster-backup-checker:latest 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/cluster-backup-checker:latest```
+4) Run the following command to push this image to the AWS ECR repository:
 
-5) Run the following command to push this image to your newly created AWS repository:
+```
+docker push 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/cluster-backup-checker:<VERSION>
+```
 
-```docker push 754256621582.dkr.ecr.eu-west-2.amazonaws.com/cloud-platform/cluster-backup-checker:latest```
+## Cluster environment and deployment
+This project requires an IAM role with the following permissions *'ec2:DescribeSnapshots'*.
 
-### Environment variables
+This script runs as a Kubernetes CronJob in the monitoring namespace on live-1 cluster. For more information on the depolyment, refer https://github.com/ministryofjustice/cloud-platform-infrastructure/blob/master/terraform/cloud-platform-components/cluster-backup-checker.tf
 
-These scripts require the following environment variables to be set.
+The following environment variables are set using terraform with *'cloud-platform-infrastructure/terraform/cloud-platform-components'* and these are passed to the script when applying terraform. Therefore you do not need to setup these.
```

Comment:
Please make the text a link to the components folder in the infrastructure github repo.